android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java

```diff
@@ -16,6 +17,10 @@ public final class WalletConstants {
             "https://doc.aurora.dev/bridge/bridge-overview/";
     public static final String URL_RAINBOW_BRIDGE_RISKS = "https://rainbowbridge.app/approvals";
 
+    // Brave
+    public static final String URL_SIGN_TRANSACTION_REQUEST =
+            "https://support.brave.com/hc/en-us/articles/4409513799693";
```

Comment:
does that link exist?

Reply:
as per https://bravesoftware.slack.com/archives/C023VS4HJ6Q/p1663666283096769 the link exists but yet to be published, not sure who and when it will be.